bench/bench.py

```diff
@@ -301,19 +303,21 @@ def decomm():
                            help='the benchmark mode (pub, pubsub)')
     tornado.options.define('commit', type=str, default='master',
                            help='the git commit')
-    tornado.options.define('golang_version', type=str, default='1.5.1',
+    tornado.options.define('golang_version', type=str, default='1.10.8',
                            help='the go version')
     tornado.options.parse_command_line()
 
     logging.getLogger('paramiko').setLevel(logging.WARNING)
     warnings.simplefilter('ignore')
 
     cmd_name = sys.argv[-1]
-    cmd_map = {
-        'bootstrap': bootstrap,
-        'run': run,
-        'decomm': decomm
-    }
-    cmd = cmd_map.get(cmd_name, bootstrap)
-
+    logging.info('CMD: %s', cmd_name)
+    cmd = bootstrap
+    if (cmd_name == 'bootstrap'):
+        cmd = bootstrap
+    if (cmd_name == 'run'):
+        cmd = run
+    if (cmd_name == 'decomm'):
```

Comment:
This part did not have to change. The restructure is OK with me, except that using parentheses like this is odd and unusual for python (they're not needed).